dashboard/app/models/ability.rb

```diff
@@ -93,6 +94,8 @@ def initialize(user)
       can :project_comments, CodeReviewComment do |_, project_owner|
         CodeReviewComment.user_can_review_project?(project_owner, user)
       end
+      can :create, ReviewableProject, user_id: user.id
```

Comment:
Who should be able to mark a project as reviewable? I think just the project owner, yeah? I think you maybe need compare the currently signed in `user` to the project owner, which you can get from the channel ID like we do here:

https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/controllers/code_review_comments_controller.rb#L68

Reply:
Ahh yes you're right - I think I misunderstood that a user would only have the option to mark their own project as reviewable but this makes more sense, will update.